### DISCUSSION TITLE
现在使用量子计算来训练模型，会有优势吗？

### DISCUSSION THREAD

**Discussion by runvyang** [PaddlePaddle/Quantum#39] [General]
如果是真实的量子计算机，主频大概是什么水平？有没有与之对应的通信（比如量子计算的结果怎么输出？）和存储（数据集怎么存）

**Comment by LeiZhang-116-4** [comment]
- 现有量子计算机的性能可以参考百度量子的[超导量子计算机](https://quantum.baidu.com/)。
- 我们可以使用经典物理设备观测量子态来获取经典信息。详情可了解[量子力学中的测量](https://en.wikipedia.org/wiki/Measurement_in_quantum_mechanics)知识。
- 经典数据用经典计算机储存即可；量子数据集可以由 [qRAM](https://arxiv.org/abs/0708.1879) 储存或获取。qRAM 目前是一个理论结果，如何实现还尚待研究。

**Comment by wangzihe1996** [comment]
量子计算机可能还没有主频的概念，有量子比特和量子体积等概念，你可以去查阅了解。
通信的话，可以通过量子测量得到经典信息。也有量子通信，不过是另外一回事了。存储，如楼上所说，有qRAM这个概念。
